Doc/whatsnew/3.14.rst

```diff
@@ -1072,6 +1072,14 @@ io
   :gh:`120754` and :gh:`90102`.)
 
 
+textwrap
+--------
+
+* Optimise the :func:`~textwrap.dedent` function, improving performance by
+  an average of 2.4x, with larger improvements for bigger inputs,
+  and fix a bug with incomplete normalization of blank lines with whitespace
```

Comment:
Maybe use two separate bullet points for that? so that the reader is able to distinguish between a performance improvement and a behavioral change.

Reply:
Well, I think it's still an improvement in some sense (even if we didn't treat it as a regular bugfix that we backport). For me I think that the behavioral change is important to note, hence I suggested using two separate bullet points (but still under the same section)